soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -0,0 +1,2005 @@
+#define H2RAMECSIE      ECReg(EC_Register_Base_Address + 0x107A)
+#define H2RAMECSA       ECReg(EC_Register_Base_Address + 0x107B)
+#define H2RAMHSS        ECReg(EC_Register_Base_Address + 0x107C)
+#define HPADR           ECReg(EC_Register_Base_Address + 0x107E)
+
+#define STCDMACR		ECReg(EC_Register_Base_Address + 0x1080)
+#define SCRA5L			ECReg(EC_Register_Base_Address + 0x1081)
+#define SCRA5M			ECReg(EC_Register_Base_Address + 0x1082)
+#define SCRA5H			ECReg(EC_Register_Base_Address + 0x1083)
+#define SCRA6L			ECReg(EC_Register_Base_Address + 0x1084)
+#define SCRA6M			ECReg(EC_Register_Base_Address + 0x1085)
+#define SCRA6H			ECReg(EC_Register_Base_Address + 0x1086)
+#define SCRA7L			ECReg(EC_Register_Base_Address + 0x1087)
+#define SCRA7M			ECReg(EC_Register_Base_Address + 0x1088)
+#define SCRA7H			ECReg(EC_Register_Base_Address + 0x1089)
+#define SCRA8L			ECReg(EC_Register_Base_Address + 0x108A)
+#define SCRA8M			ECReg(EC_Register_Base_Address + 0x108B)
+#define SCRA8H			ECReg(EC_Register_Base_Address + 0x108C)
+#define SCRA9L			ECReg(EC_Register_Base_Address + 0x108D)
+#define SCRA9M			ECReg(EC_Register_Base_Address + 0x108E)
+#define SCRA9H			ECReg(EC_Register_Base_Address + 0x108F)
+#define SCRA10L			ECReg(EC_Register_Base_Address + 0x1090)
+#define SCRA10M			ECReg(EC_Register_Base_Address + 0x1091)
+#define SCRA10H			ECReg(EC_Register_Base_Address + 0x1092)
+#define SCRA11L			ECReg(EC_Register_Base_Address + 0x1093)
+#define SCRA11M			ECReg(EC_Register_Base_Address + 0x1094)
+#define SCRA11H			ECReg(EC_Register_Base_Address + 0x1095)
+#define SCRA12L			ECReg(EC_Register_Base_Address + 0x1096)
+#define SCRA12M			ECReg(EC_Register_Base_Address + 0x1097)
+#define SCRA12H			ECReg(EC_Register_Base_Address + 0x1098)
+#define ROMARL          ECReg(EC_Register_Base_Address + 0x1099)
+#define ROMARM          ECReg(EC_Register_Base_Address + 0x109A)
+#define ROMARH          ECReg(EC_Register_Base_Address + 0x109B)
+#define SEMBARL         ECReg(EC_Register_Base_Address + 0x109C)
+#define SEMBARM         ECReg(EC_Register_Base_Address + 0x109D)
+#define SEMBARH         ECReg(EC_Register_Base_Address + 0x109E)
+#define SCRATH_SRAM     0x08
+
+
+/**
+ *
+ * (11xxh) Interrupt controller (INTC)
+ *
+ */
+#define MAX_ISR_REG_NUM	24
+#define ISR0			ECReg(EC_Register_Base_Address + 0x3F00)
+#define ISR1			ECReg(EC_Register_Base_Address + 0x3F01)
+#define ISR2			ECReg(EC_Register_Base_Address + 0x3F02)
+#define ISR3			ECReg(EC_Register_Base_Address + 0x3F03)
+#define ISR4			ECReg(EC_Register_Base_Address + 0x3F14)
+#define ISR5			ECReg(EC_Register_Base_Address + 0x3F18)
+#define ISR6			ECReg(EC_Register_Base_Address + 0x3F1C)
+#define ISR7			ECReg(EC_Register_Base_Address + 0x3F20)
+#define ISR8			ECReg(EC_Register_Base_Address + 0x3F24)
+#define ISR9            ECReg(EC_Register_Base_Address + 0x3F28)
+#define ISR10           ECReg(EC_Register_Base_Address + 0x3F2C)
+#define ISR11           ECReg(EC_Register_Base_Address + 0x3F30)
```

Comment:
The indentation in this file is off. line 311 uses tabs while line 320 uses spaces. Please be consistent (use tabs please) and make that align